drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/handlers/CommonEditHandler.java

```diff
@@ -135,7 +135,7 @@ protected static Integer getUiHeaderRowIndexLocal(ScenarioGrid scenarioGrid, Poi
     }
 
     // Indirection add for test
-    protected static boolean isEditableHeaderLocal(GridColumn<?> scenarioGridColumn, Integer uiHeaderRowIndex) {
+    protected static boolean isEditableHeaderLocal(ScenarioGridColumn scenarioGridColumn, Integer uiHeaderRowIndex) {
```

Comment:
Why have you changed this method? It seems unrelated to the fix

Reply:
It is not strictly related to the correction, you are right, I modified it simply because the chain of calls refers to ScenarioGridColumn in all points except this. What do you think?

Reply:
Fine for me, I just asked because at the end this change has no real impact yet :)

Reply:
@danielezonca 
I've asked @yesamer to do this little modifications (not strictly related to jira issue) to slowly improve the overall base code.